docs/index.rst

```diff
@@ -25,7 +25,7 @@ Pub/Sub support                     Yes
 Sentinel support                    Yes [1]_
 Redis Cluster support               WIP
 Trollius (python 2.7)               No
-Tested CPython versions             `3.5, 3.6 <travis_>`_ [2]_
```

Comment:
I'm not familiar with SphinX and I'd also argue that it's a bit out of scope of this PR ;)
But yeah, it would be a great way to ensure both resources are in sync.